### PR TITLE
feat(fuselage): new set of `SidePanel` components

### PR DESCRIPTION
**Pull request by juliajforesti**
[SIDE-4](https://rocketchat.atlassian.net/browse/SIDE-4)

## Further comments

[SIDE-4]: https://rocketchat.atlassian.net/browse/SIDE-4?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by subhamkumarr — APPROVED** [review]

File: packages/fuselage/src/components/SidePanel/SidePanelListItem.tsx
Comment (dougfabris) [inline packages/fuselage/src/components/SidePanel/SidePanelListItem.tsx:1]: missing the ref

File: packages/fuselage/src/components/SidePanel/SidePanelList.tsx
Comment (dougfabris) [inline packages/fuselage/src/components/SidePanel/SidePanelList.tsx:1]: missing the ref

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: packages/fuselage/src/components/SidePanel/SidePanelListItem.tsx
Comment (juliajforesti) [reply packages/fuselage/src/components/SidePanel/SidePanelListItem.tsx:1]: 🤡

File: packages/fuselage/src/components/SidePanel/SidePanelList.tsx
Comment (juliajforesti) [reply packages/fuselage/src/components/SidePanel/SidePanelList.tsx:1]: 🤡

**Review by dougfabris — APPROVED** [review]
LGTM!

**Pull request merged**